SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialActorChannel.h

```diff
@@ -132,6 +132,9 @@ class SPATIALGDK_API USpatialActorChannel : public UActorChannel
 	
 	void UpdateShadowData();
 
+	// If this actor channel is responsible for creating a new entity, this will be set to true once the entity is created.
+	bool bCreatedNewEntity;
```

Comment:
Public variable? Please Sahil.

Reply:
I think I'm going to leave this since another function is just clutter